documentation/docs/configuration.md

```diff
@@ -1,32 +1,38 @@
 # Configuration
 
-Configure your project through a yml-file, which is located at `.pipeline/config.yml` in the **master branch** of your source code repository.
+Configure your project through a yml-file, which is located at `.pipeline/config.yml` in the **master branch** of your
+source code repository.
```

Comment:
Please do not introduce unnecessary line breaks as this will look ugly in the docs.